cellprofiler/modules/unmixcolors.py

```diff
@@ -214,43 +214,43 @@ def add_image(self, can_remove=True):
-| %(CHOICE_AZOCARMINE)s                        |Plasma                                            |
-+----------------------------------------------+--------------------------------------------------+
-| %(CHOICE_DAB)s                               |Peroxisomes, mitochondria                         |
-+----------------------------------------------+--------------------------------------------------+
-| %(CHOICE_EOSIN)s                             |Elastic, collagen and reticular fibers            |
-+----------------------------------------------+--------------------------------------------------+
-| %(CHOICE_FAST_RED)s                          |Nuclei                                            |
-+----------------------------------------------+--------------------------------------------------+
-| %(CHOICE_FAST_BLUE)s                         |Myelin fibers                                     |
-+----------------------------------------------+--------------------------------------------------+
-| %(CHOICE_FEULGEN)s                           |DNA                                               |
-+----------------------------------------------+--------------------------------------------------+
-| %(CHOICE_HEMATOXYLIN)s                       |Nucleic acids, endoplasmic reticulum              |
-+----------------------------------------------+--------------------------------------------------+
-| %(CHOICE_HEMATOXYLIN_AND_PAS)s               |Nucleus (stained with both Hematoxylin and PAS)   |
-+----------------------------------------------+--------------------------------------------------+
-| %(CHOICE_METHYL_BLUE)s                       |Collagen                                          |
-+----------------------------------------------+--------------------------------------------------+
-| %(CHOICE_METHYL_GREEN)s                      |Chromatin                                         |
-+----------------------------------------------+--------------------------------------------------+
-| %(CHOICE_METHYLENE_BLUE)s                    |Nuclei                                            |
-+----------------------------------------------+--------------------------------------------------+
-| %(CHOICE_ORANGE_G)s                          |Erythrocytes, pancreas, pituitary                 |
-+----------------------------------------------+--------------------------------------------------+
-| %(CHOICE_PAS)s                               |Glycogen, carbohydrates                           |
-+----------------------------------------------+--------------------------------------------------+
-| %(CHOICE_PONCEAU_FUCHSIN)s                   |Red counterstain for Masson’s trichrome           |
-+----------------------------------------------+--------------------------------------------------+
++-----------------------------------+--------------------------------------------------+
```

Comment:
Please put a newline between "The stains are" and the start of the table or it won't render properly.